ops-catalog/schema-inference.flow.yaml

```diff
@@ -1,21 +1,35 @@
 collections:
   ops.us-central1.v1/inferred-schemas/L1:
+    schema: schema-inference.schema.yaml
+    key: [/collection_name]
     derive:
       using:
         sqlite: {}
       transforms:
         - name: logs
           source:
             name: ops.us-central1.v1/logs
+            partitions:
+              include:
+                kind:
+                  - capture
+              exclude:
+                name:
+                  # Don't read our own inferences.
+                  - ops.us-central1.v1/inferred-schemas/L1
+                  - ops.us-central1.v1/inferred-schemas/L2
+
+                  # NOTE(johnny): In production, I've temporarily held back
```

Comment:
You've held back some captures with lots of logs I'm guessing?

Reply:
Exactly, lots of logs that go back months.